ani-cli

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-version_number="4.4.5"
+version_number="4.4.6"
```

Comment:
Let it be a major bump so downstream updates as well.

Reply:
on it